pkg/controller/ingress/ingress_controller_test.go

```diff
@@ -44,6 +44,7 @@ var (
 			UID:         uid,
 			Labels:      map[string]string{serving.RouteNamespaceLabelKey: namespace, serving.RouteLabelKey: name},
 			Annotations: map[string]string{networking.IngressClassAnnotationKey: network.IstioIngressClassName},
+			Finalizers:  []string{"ingresses"},
```

Comment:
Can we enhance the test to actually check that the SMMR has been updated and the finalizer removed?